test/src/index-test.js

```diff
@@ -81,47 +81,47 @@ describe('tags', function () {
 		assert.equal(subtags[2].type(), 'extlang');
 		assert.equal(subtags[2].format(), 'mdl');
 
-		subtags = tags.search('Gibberish');
+		subtags = search('Gibberish');
 		assert.deepEqual(subtags, []);
 	});
 
 	it('search() puts exact match at the top', function() {
 		var subtags;
 
-		subtags = tags.search('Dari');
+		subtags = search('Dari');
 		assert(subtags.length > 0);
 
 		assert.equal(subtags[0].type(), 'language');
 		assert.equal(subtags[0].format(), 'prs');
 	});
 
 	it('subtags() returns subtags', function() {
-		var subtags;
+		var subtags2;
```

Comment:
Here and also below the top-level `subtags` function has an unlucky collision with an existing variable. Happy to consider better alternatives for variable names or function aliases.